elixir/apps/web/lib/web/controllers/error_html/500.html.heex

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=client-width, initial-scale=1" />
+    <link rel="icon" href={~p"/favicon.ico"} sizes="any" />
+    <link rel="icon" href={~p"/images/favicon.svg"} type="image/svg+xml" />
+    <link rel="apple-touch-icon" href={~p"/images/apple-touch-icon.png"} />
+    <link rel="manifest" href={~p"/site.webmanifest"} />
+    <meta name="theme-color" content="#331700" />
+    <meta name="csrf-token" content={get_csrf_token()} />
+    <title>500 Error</title>
+    <link
+      phx-track-static
+      rel="stylesheet"
+      nonce={@conn.private.csp_nonce}
+      href={~p"/assets/app.css"}
+    />
+    <script
+      defer
+      phx-track-static
+      type="text/javascript"
+      nonce={@conn.private.csp_nonce}
+      src={~p"/assets/app.js"}
+    >
+    </script>
+  </head>
+  <body class="bg-neutral-50">
+    <div class="flex items-center h-screen p-16 bg-gray-50">
+      <div class="container mx-auto flex flex-col items-center">
+        <div class="flex flex-col gap-6 max-w-md text-center">
+          <img src="/images/logo.svg" class="mr-5 h-32" alt="Firezone Logo">
+          <h2 class="font-extrabold text-9xl text-primary-600">
+            <span class="sr-only">Error</span>500
+          </h2>
+          <p class="text-2xl md:text-3xl">Something went wrong.  We'll get it fixed as soon as possible.</p>
```

Comment:
```suggestion
          <p class="text-2xl md:text-3xl">Something went wrong. We've already been notified and will get it fixed as soon as possible.</p>
```

Reply:
just to hopefully save a few support tickets

Reply:
👍 Updated the PR with your suggestion